guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,83 @@
++
+If you set a wildcard value `*` for the certificate’s Common Name `CN =` in the `/root/{smart-proxy-context}_cert/openssl.cnf` configuration file, you must add the `-t {smart-proxy-context}` option to the `katello-certs-check` command.
+
+. Use the following `{certs-generate}` commands to generate the certificate archive file for your {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+{certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar  "_/root/<certificate dir>/{smartproxy-example-com}-certs.tar_" \
+--server-cert "_/root/<certificate dir>/New-{smart-proxy-context}_cert.pem_" \
+--server-key "_/root/<certificate dir>/{smart-proxy-context}_cert_key.pem_" \
+--server-ca-cert "_/root/<certificate dir>/ca_cert_bundle.pem_" \
+--certs-update-server
+----
+
+. Retain a copy of the `{foreman-installer}` command that the `{certs-generate}` command returns for deploying the certificate to your {SmartProxyServer}.
+. On {SmartProxyServer}, to deploy the certificate, enter the `{foreman-installer}` command that the {certs-generate} command returns.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+{installer-scenario-smartproxy} \
+--foreman-proxy-content-parent-fqdn "{foreman-example-com}" \
+--foreman-proxy-register-in-foreman "true" \
+--foreman-proxy-foreman-base-url \ "https://{foreman-example-com}"
+--foreman-proxy-trusted-hosts \ "{foreman-example-com}" \
+--foreman-proxy-trusted-hosts "{smartproxy-example-com}" \
+--foreman-proxy-oauth-consumer-key "<secret>" \
+--foreman-proxy-oauth-consumer-secret "<secret>" \
+--foreman-proxy-content-certs-tar "_/root/{smartproxy-example-com}-certs.tar_" \
+--puppet-server-foreman-url "https://{foreman-example-com}" \
+--certs-update-server
+----
++
+When network connections or ports to {Project} are not yet open, you can set the `--foreman-proxy-register-in-foreman` option to _false_ to prevent {SmartProxy} from attempting to connect to {Project} and reporting errors.
+Run the installer again with the above option set to _true_ when the network and firewalls are correctly configured.
+
+. On {ProjectServer}, copy the certificate archive file to your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+scp _/root/{smart-proxy-context}_cert/{smartproxy-example-com}-certs.tar_ \
+----
+
+. On {SmartProxyServer}, to deploy the certificate, enter the `{foreman-installer}` command that the `{certs-generate}` command returns.
++
+If network connections or ports to {Project} are not yet open, you can set the `--foreman-proxy-register-in-foreman` option to _false_ to prevent {SmartProxy} from attempting to connect to {Project} and reporting errors.
+Run the installer again with this option set to _true_ when the network and firewalls are correctly configured.
+
+[IMPORTANT]
+====
+Do not delete the certificate archive file after you deploy the certificate.
+It is required, for example, when upgrading {SmartProxyServer}.
+====
+
+[NOTE]
```

Comment:
For both Satellite and Capsule sections, I think we should put these notes at the top. Changing the CA is a big deal and users should be `very` aware of that before heading down this path.

Reply:
Addressed :+1: